src/services/exchanges/base/base.exchange.service.ts

```diff
@@ -274,7 +283,7 @@ export abstract class BaseExchangeService {
     }
   };
 
-  openOrder = async (account: Account, trade: Trade): Promise<Order> => {
+  createOrder = async (account: Account, trade: Trade): Promise<Order> => {
```

Comment:
My thinking here was to distinguish between "open position" and "open order".

Reply:
yes why not :)